swap/src/alice/swap.rs

```diff
@@ -429,6 +429,10 @@ pub async fn run_until(
                 state3,
                 encrypted_signature,
             } => {
+                // TODO: Evaluate if it is correct for Alice to Redeem no matter what.
```

Comment:
True, there can be a race condition between the redeem and cancel transaction after T1.
Bob learns the secret to redeem Monero if he sees the transaction in the mempool and then can refund Bitcoin if he can race to get the cancel tx in.

I propose to solve this todo with part 2 of this plan: https://lists.comit.network/pipermail/comit-dev/2020-December/000031.html